x-pack/plugins/maps/public/actions/data_request_actions.ts

```diff
@@ -197,7 +197,7 @@ function endDataLoad(
   layerId: string,
   dataId: string,
   requestToken: symbol,
-  data: FeatureCollection | object,
+  data: object,
```

Comment:
Could you make corresponding change here?

https://github.com/elastic/kibana/blob/d9b10e233a5758c8a5a5e9ff713eee0b7fe36a7d/x-pack/plugins/maps/public/actions/data_request_actions.ts#L48